components/clp-py-utils/clp_py_utils/clp_config.py

```diff
@@ -301,7 +304,8 @@ def validate_logging_level(cls, field):
 
 
 class CLPConfig(BaseModel):
-    execution_container: str = "ghcr.io/y-scope/clp/clp-execution-x86-ubuntu-focal:main"
+    execution_container: typing.Optional[str]
+    _os_release_file_path: pathlib.Path = PrivateAttr(default=OS_RELEASE_FILE_PATH)
```

Comment:
Based on our guidelines, `_os_release_file_path` is private and should be after all the public members (before the methods though).